docs/configuration.md

```diff
@@ -942,6 +942,210 @@ editing the `conf` file in a text editor. Use the examples as reference.
     </tr>
 </table>
 
+### dd_configuration_option
+
+<table>
+    <tr>
+        <td>Description</td>
+        <td colspan="2">
+            @warning{Windows only!}
+            Perform additional configuration for the display device:
+            <ul>
+                <li>`disabled` - perform no additional configuration (disables all `dd_` configuration options).</li>
+                <li>`verify_only` - verify that display is active only (required for changing display mode and other options).</li>
+                <li>`ensure_active` - activate the display if it's currently inactive.</li>
+                <li>`ensure_primary` - activate the display if it's currently inactive and make it primary.</li>
+                <li>`ensure_only_display` - activate the display if it's currently inactive and disable all others.</li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <td>Default</td>
+        <td colspan="2">@code{}verify_only@endcode</td>
+    </tr>
+    <tr>
+        <td>Example</td>
+        <td colspan="2">@code{}
+            dd_configuration_option = ensure_only_display
+            @endcode</td>
+    </tr>
+</table>
+
+### dd_resolution_option
+
+<table>
+    <tr>
+        <td>Description</td>
+        <td colspan="2">
+            @warning{Windows only!}
+            @note{"Optimize game settings" must be enabled for this option to work.}
+            Perform additional resolution configuration for the display device:
+            <ul>
+                <li>`disabled` - perform no additional configuration.</li>
+                <li>`automatic` - change resolution to the requested resolution from the client.</li>
+                <li>`manual` - change resolution to the user specified one (set via `dd_manual_resolution`).</li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <td>Default</td>
+        <td colspan="2">@code{}automatic@endcode</td>
```

Comment:
```suggestion
        <td colspan="2">Automatic</td>
```

> [!NOTE]
> Applies in a few places.

Reply:
My intentions was to provide and actual value in a code block. Do you suggest to remove the code block or to also make it a description (in a sense), because you capitalized the first letter?